google-cloud-pom-parent/versions.txt

```diff
@@ -1,4 +1,4 @@
 # Format:
 # module:released-version:current-version
 
-google-cloud-pom-parent:0.0.0:1.0.0-SNAPSHOT
+google-cloud-pom-parent:0.1.0:0.1.0
```

Comment:
This google-cloud-pom-parent/versions.txt is updated. But the occurrences of `{x-version-update:google-cloud-pom-parent:current}` is not updated.

<img width="1232" alt="Screen Shot 2022-10-13 at 3 28 24 PM" src="https://user-images.githubusercontent.com/28604/195699209-36266112-c0e4-4e36-9d58-7f2fd030b53d.png">